src/xo-app/host/tab-patches.js

```diff
@@ -148,26 +166,20 @@ export default class HostPatches extends Component {
               icon='host-reboot'
               labelId='rebootUpdateHostLabel'
             />}
-            {isEmpty(missingPatches)
-              ? <TabButton
-                disabled
-                handler={installAllPatches}
-                icon='success'
-                labelId='hostUpToDate'
-              />
-              : <TabButton
-                btnStyle='primary'
-                handler={installAllPatches}
-                icon='host-patch-update'
-                labelId='patchUpdateButton'
-              />
-            }
+            <TabButton
+              disabled={isMissingPatchesEmpty}
+              btnStyle={!isMissingPatchesEmpty && 'primary'}
```

Comment:
Ok, that was an issue, rebase your branch on `next-release` and you'll see the warning.